drive/types.bal

```diff
@@ -123,76 +123,76 @@ public type FileMetadata record {
 #             If not specified as part of a copy request, 
-#             the file will inherit any discoverable parents of the source file. 
+#             the file will inherit any discoverable parents of the source file
 #             Update requests must use the addParents and removeParents parameters to modify the parents list. 
 # + headRevisionId - The ID of the file's head revision. This is currently only available for files with binary content 
-#                    in Google Drive.  
-# + modifiedByMeTime - The last time the file was modified by the user (RFC 3339 date-time). 
-# + modifiedByMe - Whether the file has been modified by this user.
-# + shared - Whether the file has been shared. Not populated for items in shared drives.  
+#                    in Google Drive
+# + modifiedByMeTime - The last time the file was modified by the user (RFC 3339 date-time)
+# + modifiedByMe - Whether the file has been modified by this user
+# + shared - Whether the file has been shared. Not populated for items in shared drives 
 # + hasAugmentedPermissions - Whether there are permissions directly on this file. 
-#                             This field is only populated for items in shared drives.    
+#                             This field is only populated for items in shared drives   
 # + trashingUser - If the file has been explicitly trashed, the user who trashed it. Only populated for items in shared 
-#                  drives.  
+#                  drives
 # + thumbnailLink - A short-lived link to the file's thumbnail, if available. Typically lasts on the order of hours. 
 #                   Only populated when the requesting app can access the file's content. 
 #                   If the file isn't shared publicly, the URL returned in Files.thumbnailLink must be fetched using 
-#                   a credentialed request.  
-# + permissionIds - List of permission IDs for users with access to this file.  
+#                   a credentialed request
+# + permissionIds - List of permission IDs for users with access to this file
 # + quotaBytesUsed - The number of storage quota bytes used by the file.  This includes the head revision as well as 
-#                    previous revisions with keepForever enabled.  
-# + lastModifyingUser - The last user to modify the file.  
+#                    previous revisions with keepForever enabled 
+# + lastModifyingUser - The last user to modify the file
 # + md5Checksum - The MD5 checksum for the content of the file. This is only applicable to files with binary content
-#                 in Google Drive.  
+#                 in Google Drive
 # + fileExtension - The final component of fullFileExtension. This is only available for files with binary content 
-#                   in Google Drive.  
+#                   in Google Drive
 # + fullFileExtension - The full file extension extracted from the name field.  May contain multiple concatenated 
```

Comment:
Do we mean the same thing by "fields" and "parameters" here? If so, it is better to stick to one term. This comment is applicable to all the files in the pull request.

Suggestion: "The full file extension extracted from the `name` field. This may contain multiple concatenated extensions, such as `tar.gz`. This s only available for files with binary content in Google Drive"

Reply:
Hi,
We have used the same content from [1] for parameter descriptions
[1]https://developers.google.com/drive/api/v3/reference/files
Thanks